modules/UI/videolayout/SmallVideo.js

```diff
@@ -217,8 +197,8 @@ SmallVideo.prototype.showAudioIndicator = function(isMuted) {
 
     if (!isMuted) {
         if (audioMutedSpan.length > 0) {
-            audioMutedSpan.popover('hide');
```

Comment:
This should be handled with the new tooltip as well. If someone is muted and they hover the muted icon and click M shortcut they will unmute and the tooltip will stay.